package.json

```diff
@@ -17,7 +17,8 @@
 	},
 	"dependencies": {
 		"axios": "^0.26.1",
-		"azure-iothub": "^1.14.4"
+		"azure-iothub": "^1.14.4",
+		"https": "^1.0.0"
```

Comment:
I don't know what this library is. https://www.npmjs.com/package/https looks fishy since there's nothing in it and no repository is linked. When I look into the node_modules it is just a single package.json without any code. You shouldn't even need to install this since https is a node built in. You should be able to just do `import https from 'https'`.

Reply:
Done